client/src/services/formatting.js

```diff
@@ -5,6 +5,17 @@ export const formatString = (string, params) => {
     return string;
 };
 
-export const getVideoIdFromLink = (videoLink) => {
-    return videoLink.split("?v=")[1];
-};
\ No newline at end of file
+export const paramsObjectToQueryString = paramsObject => {
+    let query = "";
+    if (paramsObject){
+        const firstParamKey = Object.keys(paramsObject)[0];
+        query += `?${firstParamKey}=${paramsObject[firstParamKey]}`;
+        delete paramsObject[firstParamKey];
+        for (const [key, value] of Object.entries(paramsObject)) {
+            query += `&${key}=${value}`;
+        }
+    }
+    return query;
+};
+
```

Comment:
I can't comment on that line but in line number 1 of this file you have
```js
export const formatString = (string, params) => {
```
and when you call the method `formatString` you do:
```js
formatString("some {} string {}", ["array of", "parameters"])
```
Which is not nice for the eye. Instead you can do:
```js
export const formatString = (string, ...params) => {
```
Notice the `...params` (which turn all the params for this method (from the second param) to an array of params)
Then you can call this method like this: 
```js
formatString("some {} string {}", "no array", "anymore")